fbpic/particles/push/cuda_methods.py

```diff
@@ -9,6 +9,47 @@
 import math
 from scipy.constants import c, e
```

Comment:
Note: I had to move this function to the beginning of the file, in order for the other functions to compile.